helpers/c#/helpers.cs

```diff
@@ -138,8 +138,26 @@ public bool isGreaterThan(object a, object b)
 
     public bool isLessThan(object a, object b)
     {
-
-        return !isGreaterThan(a, b) && !isEqual(a, b);
+        if (a.GetType() == typeof(Int64))
```

Comment:
@ttodua ok I agree that `isLessThan(a,b)` is not necessarily the same as `!isGreaterThan` because c# does not have falsy values. However, this code will break if a or b = null.

Reply:
yeah, right, it needs to be checked against null at first place, but this is not isolated to `iLT`. my note was just that `iLT` should not be `! iGT` . however, as you say modification also needs for `iGT` too. 👍